lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -99,6 +99,18 @@ aaa_server_group:
     default_value: "default"
 
 dnsclient:
+  domain:
+    config_get: "show running-config all"
+    config_get_token: '/^ip domain-list (\S+)$/'
+    config_set: '<state> ip domain-list <name>'
+    default_value: ''
+
+  domain_vrf:
+    config_get: "show running-config all"
+    config_get_token: ['/vrf context %s/', '/ip domain-list (\S+)/']
```

Comment:
Hmm.  We do this for ospf_vrf.
```
vrf:
    config_get_token: ['/^router ospf <name>$/i', '/^vrf\s+(\S+)$/']
    config_set: ["router ospf <name>", "<state> vrf <vrf>", "end"]

vrf_ids = config_get('ospf', 'vrf', name: name)
```
What does your call to config_get look like?  I suggested you use `@vrf` but in this case since it's in the class method it should be just `vrf`